README.md

```diff
@@ -23,6 +23,18 @@ You can reach the maintainers of this project at:
 
 Check the [Development Doc](docs/development.md) for how to contribute to the repo.
 
+## How to Deploy
+### Deploy Hub
+1. Run `make deploy-hub`
+
+### Deploy Spoke
+1. Create a bootstrap secret in the namespace of `open-cluster-management` with the following command:
+
+    ```sh
```

Comment:
We need to decode the secret and replace the URL with the kubernetes service cluster IP before `make deploy-spoke`.

I wouldn't mind having a make target in the future that makes the secret.